tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -130,7 +143,21 @@ private[scalanative] object LLVM {
         asan(config) ++ target(config)
     }
     val paths = objectsPaths.map(_.abs)
-    val compile = config.clangPP.abs +: (flags ++ paths ++ linkopts)
+    val llvmLinkInfo = flags ++ paths ++ linkopts
+    def dump(strings: Seq[String], fileName: String): Unit = {
```

Comment:
I remember it's a fix for passing too many file paths to the clang compiler. It would be useful to  add a comment why are we creating this file. 
As there is just a single usage we can inline it and eventually wrap into braces of `locally{...}` block to not polute namespace.